projects/core/src/auth/facade/auth.service.ts

```diff
@@ -69,6 +68,27 @@ export class AuthService {
     this.store.dispatch(new LoadClientToken());
   }
 
+  /**
+   * Returns the client token
+   */
+  getClientToken(): Observable<ClientToken> {
```

Comment:
please check the logic of `getClientToken` and `refreshClientToken`. They looks very similar. And please add more accurate doc comment for each function.

Reply:
They're a bit different: the `refreshClientToken` is checking for an existing token, and if there’s none it load a new one